### PR TITLE
Adding docblock for tribe_events_timezone_choice()

### PR DESCRIPTION
**Pull request by deblynprado**

### CODE REVIEW

**Review by redscar — COMMENTED** [review]
Looking good, the `apply_filters` also needs it's own docblock.

**Comment by deblynprado** [comment]
@redscar just added :D

File: src/functions/template-tags/date.php [hunk 1]
```diff
@@ -106,24 +106,34 @@ function tribe_event_is_on_date( $date = null, $event = null ) {
...
+	 * This function provides an opportunity to modify the timezone `<option>` elements used within the timezone picker.
+	 * It utilizes the `wp_timezone_choice` function to generate the timezone options and allows for further customization
+	 * through the `tribe_events_timezone_choice` filter.
 	 *
 	 * @since 4.6.5
 	 *
-	 * @param string $selected_zone
-	 * @param string $locale (optional)
+	 * @param string      $selected_zone The selected timezone.
+	 * @param string|null $locale Optional. The locale to use for the timezone choice. Default null.
```
Comment (codingmusician) [inline src/functions/template-tags/date.php:118]: Description text should line up, too.
```suggestion
	 * @param string      $selected_zone The selected timezone.
	 * @param string|null $locale        Optional. The locale to use for the timezone choice. Default null.
```

**Comment by deblynprado** [comment]
@codingmusician I committed the changes you suggested. 
Let me know if I'm doing it wrong

**Review by codingmusician — DISMISSED** [review]
Looks great!

**Comment by bordoni** [comment]
Move to `release/T24.babayaga`

**Pull request closed without merging**